app/views/foreman_webhooks/webhook_templates/remote_execution_-_host_job.erb

```diff
@@ -18,4 +18,3 @@ model: WebhookTemplate
 # Task ended at <%= @object.task.ended_at %>
 # Task resulted with <%= @object.task.result %>
 # Task state <%= @object.task.state %>
-# Task action output <%= @object.task.action_output %>
```

Comment:
I've found why it is not available anymore: https://github.com/theforeman/foreman-tasks/commit/b904dcfad3878365177e644cf8c8caf7b05cf0f5#diff-faa1c0e41ac8cbfd311a3e099f6da63d947a60f226b8227ccc6b3637601971c2

We need to rename it. Also, note that there are more templates in the repo that use `action_output`.

Reply:
Also, if we want to not remove it then it worth to document that method as well (in foreman-tasks plugin)

Reply:
Good catch, I think I will close this PR and create an alias in foreman-tasks.